### PR TITLE
feat(desktop): support preview file at new window

### PR DESCRIPTION
**Pull request by BlackHole1**
Demo video:

https://user-images.githubusercontent.com/8198408/147338748-9b4fe003-eca3-47b6-b733-8c074aa73b36.mp4

**No memory leaks**

### CODE REVIEW

File: desktop/renderer-app/src/pages/CloudStoragePage/CloudStorageFilePreview/DynamicPreview.less [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .dynamic-preview-container {
-    height: 88vh;
+    height: 99.9%;
```
Comment (Cyberhan123) [inline desktop/renderer-app/src/pages/CloudStoragePage/CloudStorageFilePreview/DynamicPreview.less:3]: why use 99.9%？What's the difference between `99.9%` and `100%`

File: packages/flat-types/src/constants/index.ts [hunk 2]
```diff
@@ -28,6 +29,11 @@ export const PageSize = {
         width: 320,
         height: 48,
     },
+    PreviewFile: {
+        // approximately equal to 16/9
+        width: 888.9,
+        height: 500,
+    },
```
Comment (Cyberhan123) [inline packages/flat-types/src/constants/index.ts]:
```suggestion
PreviewFile: {
        // approximately equal to 16/9
        width: 889,
        height: 500,
    },
```

Comment (BlackHole1) [reply packages/flat-types/src/constants/index.ts]: You are right
图片

File: desktop/renderer-app/src/pages/CloudStoragePage/CloudStorageFilePreview/DynamicPreview.less (hunk 1 above)
Comment (BlackHole1) [reply desktop/renderer-app/src/pages/CloudStoragePage/CloudStorageFilePreview/DynamicPreview.less:3]: **99.9%**
图片

**100%**
图片

when height is `100%`, scroll bar will appear.

Comment (BlackHole1) [reply desktop/renderer-app/src/pages/CloudStoragePage/CloudStorageFilePreview/DynamicPreview.less:3]: cc @Cyberhan123

**Review by Cyberhan123 — DISMISSED** [review]

**Review by Cheerego7 — DISMISSED** [review]

**Review by Cheerego7 — APPROVED** [review]

**Pull request merged**